File: ZooZMobileWebPHPLib/lib/invoice.item.php
<?php
	class InvoiceItem {
		
		public $name;
		public $id;
		public $quantity;
		public $price;
		public $taxAmount;
		public $additionalDetails; 
		
		function __construct($name, $id, $quantity, $price, $taxAmount, $additionalDetails) {
			$this->name = $name;
			$this->id = $id;
			$this->quantity = $quantity;
			$this->price = $price;
			$this->taxAmount = $taxAmount;
			$this->additionalDetails = $additionalDetails;
		}
	}
?>

File: ZooZExtendedServerAPIJavaLib/src/com/zooz/extended/java/lib/model/InvoiceItem.java
package com.zooz.extended.java.lib.model;

import org.json.JSONException;
import org.json.JSONObject;

import com.zooz.extended.java.lib.utils.ZooZJSONUtils;

public class InvoiceItem {
	
	private String name;
	private String id;
	private double quantity;
	private double price;
	private double taxAmount;
	private String additionalDetails;
	
	
	public InvoiceItem(JSONObject jsonObject) throws JSONException {
		this.name = ZooZJSONUtils.getDecodedString(jsonObject, "name");
		this.id = ZooZJSONUtils.getString(jsonObject, "id");
		this.quantity = ZooZJSONUtils.getDouble(jsonObject, "quantity");
		this.price = ZooZJSONUtils.getDouble(jsonObject, "price");
		this.taxAmount = ZooZJSONUtils.getDouble(jsonObject, "taxAmount");
		this.additionalDetails = ZooZJSONUtils.getDecodedString(jsonObject, "additionalDetails");
	}


	public String getName() {
		return name;
	}


	public String getId() {
		return id;
	}


	public double getQuantity() {
		return quantity;
	}


	public double getPrice() {
		return price;
	}


	public double getTaxAmount() {
		return taxAmount;
	}


	public String getAdditionalDetails() {
		return additionalDetails;
	}

	
}
